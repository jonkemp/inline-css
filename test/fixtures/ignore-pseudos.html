<html><link rel="stylesheet" href="ignore-pseudos.css" /><body>
<a href="#">Test</a></body></html>
