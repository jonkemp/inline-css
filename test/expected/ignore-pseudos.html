<html><head></head><body>
<a href="#" style="text-decoration: underline;">Test</a>
</body></html>
